### PR TITLE
Create blog post for v1.35

### LINKED ISSUE #388: Blog post for v1.35.0
I've got a draft for this release's blog post going and will put it up for review soon. I'd welcome feedback for what to say.

### PR DESCRIPTION
**Pull request by DNin01**
Resolves #388

To be merged when the new version becomes available to users.

- [x] Date

### CODE REVIEW

File: content/blog/v1-35-released.md
Comment (WorldLanguages) [inline content/blog/v1-35-released.md:4]: Date TODO

**Comment by CST1229** [comment]
It might be worth noting somewhere that the slow messaging "bug" was also fixed, maybe not as a new section but in the last paragraph.

**Comment by Secret-chest** [comment]
> You may have noticed the messenger recently started loading very slowly. With this update, Messaging is about as fast as before.

**Comment by WorldLanguages** [comment]
Here's the full changelog
```
- New addon: skewing in costume editor
- New addon: preview project instructions and notes [beta]
- New addon: point towards random direction block
- New addon: sticky footer (previously part of "infinite scrolling")
- Scratch Addons now shows an error page if settings storage is failing
- "Scratch Messaging" addon: enhanced loading times
- "Customizable block text style" addon: new "text size" setting
- "Fix sprite pane glitching" addon: it is now optional to disable sprite wobble
- "Project video recorder" addon: increased maximum recording time to 10 minutes
- "Arrow key incrementation in editor" addon: now supports more number fields and allows for custom values
- "Insert blocks by name" addon: fixed multiple bugs, enhanced performance and autocomplete
- "Username search" addon: now shows the user's profile picture
- "Customizable block text style" addon: general fixes
- "Scratch 2.0 → 3.0" addon: improved the look of tabs on the My Stuff page
- "TurboWarp button" addon: improved button contrast
- "Website dark mode and customizable colors" addon: better support for Scratch statistics page
- "Old studio layout" addon: no longer changes the aspect ratio of studio thumbnails
- Bug fix: "auto-hiding block palette" deleting blocks unexpectedly when dropping them close to the border
- Bug fix: "save blocks as image" cropping define blocks incorrectly
- Bug fix: sprite pane not using all available space if both "compact editor" and "collapsing sprite properties" enabled
- Bug fix: untranslated text in the Scratch editor menu bar if "customizable block colors" enabled
- Bug fix: "editor dark mode and customizable colors" showing incorrect icons in the costume editor
- Bug fix: "confirmations" recognizing the "choose a tutorial" button as a Share button
- Bug fix: "better forum quoter" sometimes quoting image URLs incorrectly
```

**Comment by mybearworld** [comment]
> > You may have noticed the messenger recently started loading very slowly. With this update, Messaging is about as fast as before.

Would it be worth mentioning that this is a thing Scratch did?

File: content/blog/v1-35-released.md
Comment (mybearworld) [inline content/blog/v1-35-released.md:46]: It would probably be good to mention that this might have some issues, something like this:

```suggestion
If you can't click links on editable fields, how are you supposed to know whether a link, mention, or hashtag in your project's Instructions or Notes and Credits section can actually be followed? Well, the new "Preview project instructions and notes" addon allows you to preview what your project page looks like to other users. Note that this addon is currently in beta, and you may experience some issues. Feel free to [leave feedback](/feedback) if you do!
```

**Comment by CST1229** [comment]
> Would it be worth mentioning that this is a thing Scratch did?

Maybe?

**Comment by DNin01** [comment]
> > You may have noticed the messenger recently started loading very slowly. With this update, Messaging is about as fast as before.
> 
> Would it be worth mentioning that this is a thing Scratch did?

Probably not. We don't need to get into detail about what was actually happening, I'd just briefly brush on the fact that the bug was fixed:

> ...fixed a few bugs like messages loading slower than usual in Scratch Messaging.

**Comment by DNin01** [comment]
> ```
> - Scratch Addons now shows an error page if settings storage is failing
> ```

I would appreciate if we could mention that briefly somehow.

**Comment by BroJac5246** [comment]
> > ```
> > - Scratch Addons now shows an error page if settings storage is failing
> > ```
> 
> I would appreciate if we could mention that briefly somehow.

I feel like that suggests that Scratch Addons is likely to crash. Really, this ideally isn't something that the average user would be aware of because they shouldn't need to be aware of it.

**Comment by DNin01** [comment]
Waiting for final decision for Arrow key incrementation customizability implementation: https://github.com/ScratchAddons/ScratchAddons/issues/6765. Will need to edit blog post if action is taken.

File: content/blog/v1-35-released.md
Comment (mybearworld) [inline content/blog/v1-35-released.md:40]: This can be removed now I think? https://github.com/ScratchAddons/ScratchAddons/issues/6765#issuecomment-1761957252

File: content/blog/v1-35-released.md
Comment (WorldLanguages) [reply content/blog/v1-35-released.md:4]:
```suggestion
date: 2023-10-13T00:00:00Z
```

**Review by WorldLanguages — APPROVED** [review]

**Comment by WorldLanguages** [comment]
Ok to merge? Firefox users are already getting 1.35.0

**Comment by DNin01** [comment]
Whenever you want, but I like merging after the release is approved in the Chrome Web Store, since that's our largest audience. It just leaves less users confused on where/when the update is.

**Comment by DNin01** [comment]
Alright! We're already receiving bug reports about this release, so we should be all ready to go.

**Pull request merged**
